pallets/subtensor/src/lib.rs

```diff
@@ -347,6 +349,17 @@ pub mod pallet {
         pub ip_type: u8, // --- Prometheus ip type, 4 for ipv4 and 6 for ipv6.
 	}
 
+	// Rate limiting
+	#[pallet::type_value]
+	pub fn DefaultTxRateLimit<T: Config>() -> u64 { T::InitialTxRateLimit::get() }
+	#[pallet::type_value]
+	pub fn DefaultLastTxBlock<T: Config>() -> u64 { 0 }
+
+	#[pallet::storage] // --- ITEM ( tx_rate_limit )
+	pub(super) type TxRateLimit<T> = StorageValue<_, u64, ValueQuery, DefaultTxRateLimit<T>>;
+	#[pallet::storage] // --- MAP ( key ) --> last_block
```

Comment:
hotkey?

Reply:
This is the storage value to allow configuration of rate limit on the network, not where we store the last tx block per key